File: 1313. Decompress Run-Length Encoded List.cpp
class Solution {
public:
    vector<int> decompressRLElist(vector<int>& nums) {
        vector <int> lst;
        for(int i=0;i<nums.size();i++){
            for(int j=0;j<nums[i];j++) lst.push_back(nums[i+1]);
            i++;
        }
        return lst;
    }
};

File: 1313. Decompress Run-Length Encoded List.py
class Solution:
    def decompressRLElist(self, nums: List[int]) -> List[int]:
        l=[]
        for i in range(0,len(nums),2):
            b=nums[i+1]
            for i in range(nums[i]):
                l.append(b)
        return l
                
